zio-http/shared/src/main/scala-2/zio/http/HandlerVersionSpecific.scala

```diff
@@ -0,0 +1,25 @@
+package zio.http
+
+import zio.{Tag, Trace, ZIO, ZLayer}
```

Comment:
```suggestion
import zio.{Tag, Trace, ZIO, ZEnvironment, ZLayer}
```

For `asInstanceOf` at line 18